src/readfish/plugins/guppy.py

```diff
@@ -62,6 +63,18 @@ def __init__(
         self.supported_basecall_models = None
         self.run_information = run_information
 
+        if self.run_information:
```

Comment:
make this a shared function between the two plugins, I know we are deprecating Guppy at some point, but we may as well not bother adding more duplicated code!
Stick it in`utils` or somethin

Reply:
So the code logic is slightly different depending on which plugin it is.

Reply:
Technically the code isn't an exact duplicate. Messages and responses to versions are specific to the caller. I could write a complex function to do this but it would then need to know which plugin has called it.